README.md

```diff
@@ -47,6 +47,7 @@ buildscript {
   }
 }
 ```
+(If you are not using Android, this whole block will go at the top of your build.gradle file, not settings.gradle!)
```

Comment:
It shouldn't go in settings.gradle on Android either.

Reply:
Right, I think the instructions are clear for how to do it in Android. but when you say things like `The plugin can then be applied as follows within your app module's build.gradle :`, it implies two build.gradle files, and since it was project/app module level, I figured I had to put it in `settings.gradle`. Putting the `buildscript` block in `settings.gradle` was the mistake I made until I figured out to put it in my normal(and only) `build.gradle`. Part of it was me being an idiot, but clarification for JVM can't hurt, no?

Reply:
Settings.gradle is never mentioned. If we need to clarify not to put it there should we also clarify to not put it in gradle.properties or somewhere else. I would suggest talking about where to put it rather than mentioning where not to put it

Reply:
'settings.gradle' never got mentioned because this looks like an android-first library and I haven't seen a 'settings.gradle' file in an android project. It isnt my library and if it continued to look android-first that would be fine, I just want to suggest clarification in the documentation for non-android users.

Reply:
As @digitalbuddha mentionning places *not* to put the plugin adds to the cognitive load so it might actually confuse users more.

settings.gradle is not related to android or non-android. I think it has been there for a long time in gradle, especially for [multi-module projects](https://guides.gradle.org/creating-multi-project-builds/)